app/Models/Question.php

```diff
@@ -36,4 +43,9 @@ public function choices(): HasMany
     {
         return $this->hasMany(Choice::class);
     }
+
+    public function getChoices()
+    {
+        return Options::forArray($this->choices()->pluck('value'), useLabelsAsValue: true)->toArray();
+    }
```

Comment:
Instead of this, choices can be fetched using:

```php
Options::forModels(
	Choice::query()->whereIn('id', $question->choices->pluck('id')->toArray()),
	label: fn(Choice $choice) => $choice->getTranslation('label', 'en')
)->toArray();
```

The `HasTranslations` trait will need to be added to the `Choice` model and the `value` column on the `Choice` model can be dropped.

Reply:
Then user choices can be compared using numeric IDs instead of strings.